### PR TITLE
Fulfill license obligations for helm generated files

### PR DESCRIPTION
**Pull request by strieflin**
Signed-off-by: Sven Trieflinger <sven.trieflinger@de.bosch.com>

### CODE REVIEW

File: charts/minio/templates/_helpers.tpl [hunk 1]
```diff
@@ -1,11 +1,9 @@
-{{/*
-Copyright (c) 2021 - for information on the respective copyright owner
-see the NOTICE file and/or the repository https://github.com/carbynestack/carbynestack.
-
-SPDX-License-Identifier: Apache-2.0
-*/}}
-
 {{/* vim: set filetype=mustache: */}}
+{{/* Copyright (c) 2021 - for information on the respective copyright owner */}}
+{{/* see the NOTICE file and/or the repository https://github.com/carbynestack/ephemeral. */}}
```
Comment (sbckr) [inline charts/minio/templates/_helpers.tpl:3]: wrong repository link -> https://github.com/carbynestack/carbynestack

**Review by sbckr — CHANGES_REQUESTED** [review]
See file comment

File: charts/minio/templates/_helpers.tpl (hunk 1 above)
Comment (strieflin) [reply charts/minio/templates/_helpers.tpl]: Fixed

**Review by sbckr — APPROVED** [review]
:+1:

**Pull request merged**